spec/acceptance/identity/specify_app_package_authority_idents_spec.rb

```diff
@@ -21,54 +21,54 @@
     target = "c:/#{SecureRandom.uuid}"
     verify_acl_command = "icacls #{target}"
 
-    windows_agents.each do |agent|
-      context "Specify APPLICATION PACKAGE AUTHORITY accounts on #{agent}" do
-        it "Check Minimum Supported OS for #{account[:id]}" do
-          kernelmajversion = on(agent, facter('kernelmajversion')).stdout.chomp.to_f
-          # try next agent if user is unsupported on this Windows version
-          if kernelmajversion < account[:minimum_kernel]
-            warn("This test requires Windows kernel #{account[:minimum_kernel]} but this host only has #{kernelmajversion}")
-            skip
+    context 'specify APPLICATION PACKAGE AUTHORITY accounts' do
+      it "Check Minimum Supported OS for #{account[:id]}" do
+        # use of host_inventory returns nil with localhost
+        kernelmajversion = run_shell('facter kernelmajversion').stdout.to_f
```

Comment:
also check this out. Was unable to get serverspec `host_inventory` to work with AppVeyor

Reply:
Do you mean `invenotory_hash`? These tests are being skipped because the kernelmajversion is being returned as "6.3" in appveyor. I'm filing a bug to have this solved in a different commit.